deliver/lib/deliver/upload_metadata.rb

```diff
@@ -183,6 +197,17 @@ def load_from_filesystem(options)
         UI.message("Loading '#{path}'...")
         options[key] ||= File.read(path)
       end
+
+      # Load review information
+      options[:app_review_information] ||= {}
+      REVIEW_INFORMATION_VALUES.values.each do |option_name|
+        path = File.join(options[:metadata_path], REVIEW_INFORMATION_DIR, "#{option_name}.txt")
+        next unless File.exist?(path)
+        next if options[:app_review_information][option_name]
```

Comment:
```ruby
next if options[:app_review_information][option_name].to_s.length > 0
```

The code sample works even if the value is `nil`, as `to_s` of `nil` is `""`. This can be one line 👍